components/shared/NavigationLink/index.js

```diff
@@ -4,7 +4,7 @@ import cn from "classnames";
 const NavigationLink = ({ url, text, linkClassName, secondaryClassName }) => {
   return (
     <li className={cn("text-center", secondaryClassName)}>
-      <Link href={url}>
+      <Link href={url} prefetch  ={false}>
```

Comment:
please use the higher-order component we discussed earlier ( I know you planned to do it tomorrow, I am just leaving this here as a reminder just in case anyone else decided to work on this component)